Source/Core/VideoBackends/Software/SWVertexLoader.cpp

```diff
@@ -88,13 +89,6 @@ void SWVertexLoader::vFlush()
   for (u32 i = 0; i < IndexGenerator::GetIndexLen(); i++)
   {
     u16 index = LocalIBuffer[i];
-
-    if (index == 0xffff)
```

Comment:
I wonder, will this break primitive_restart on video sw?

Reply:
We set SupportsPrimitiveStart to false in sw. Checked that before removing those lines :)